### PR TITLE
Automation/playwright

### PR DESCRIPTION
**Pull request by alvitazwar**

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# e2e test for transcoder
+
+name: CI for Transcoder
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: 
+      - master
```
Comment (pooja-muchandikar) [inline .github/workflows/e2e.yml:10]: @alvitazwar Add Develop branch as well, as tests should run when PR is raised against develop branch also.

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      
+  pull_request:
+    branches: 
+      - master
```
Comment (pooja-muchandikar) [inline .github/workflows/e2e.yml:14]: Same here as well add develop branch

File: tests/e2e-playwright/bin/pr-status.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    auth: process.env.TOKEN,
+});
+
+octokit.request("POST /repos/{org}/{repo}/statuses/{sha}", {
+    org: "rtCamp",
+    repo: "transcoder",
+    sha: process.env.SHA ? process.env.SHA : process.env.COMMIT_SHA,
+    state: "success",
+    conclusion: "success",
+    target_url:
```
Comment (pooja-muchandikar) [inline tests/e2e-playwright/bin/pr-status.js:17]: @alvitazwar Change the target URL it should redirect to the [results page](https://github.com/rtCamp/transcoder/blob/84de83bb8997579576638b43934a0f1bb68ebed9/tests/e2e-playwright/bin/pr-status.js#L17).

**Review by pooja-muchandikar — COMMENTED** [review]
@alvitazwar 
- Remove Uploads folder if images are not used anywhere
- Remove artifacts folder

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (alvitazwar) [reply .github/workflows/e2e.yml:10]: Done

**Review by pooja-muchandikar — APPROVED** [review]

**Pull request merged**